tests/fv3_conf/control_run.IN

```diff
@@ -24,6 +24,17 @@ else
     SUFFIX=${BL_SUFFIX}
   fi
 
+  # CMEPS restart and pointer files
```

Comment:
I'm worried that this may cause unbound variable errors when running non-cmeps, non-wave tests. Have you run any of the standalone tests (control_p8 etc) with this change?

Reply:
Yes I knew that it might cause an issue, I just wanted to ask Jun in our meeting today if she prefers either to have a new atmw_control_run.IN file or I include an if condition for that. I modified the control_run.IN by adding an if condition, please check it out. Thanks, Denise.

Reply:
Yes, it is.